runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -75,13 +115,25 @@ ClassSerializationRecord::ClassSerializationRecord(uintptr_t id, uintptr_t class
    memcpy(_name, J9UTF8_DATA(J9ROMCLASS_CLASSNAME(romClass)), _nameLength);
    }
 
+ClassSerializationRecord::ClassSerializationRecord() :
+   AOTSerializationRecord(0, 0, AOTSerializationRecordType::Class),
+   _classLoaderId(0), _hash(NULL), _romClassSize(0),
```

Comment:
The `_hash` constructor cannot be given a `NULL` argument - this causes a segfault. This should be `_hash()` instead, or removed entirely.

Reply:
`_hash()` would be correct.